src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -692,56 +705,75 @@
           }
         }
       },
-      "required": [ "tool" ]
+      "required": [ "name" ]
     },
-    "tool": {
-      "description": "The analysis tool that was run.",
+
+    "invocation": {
+      "description": "The runtime environment of the analysis too run.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
-        "name": {
-          "description": "The name of the tool.",
+        
+        "parameters": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
           "type": "string"
         },
 
-        "fullName": {
-          "description": "The name of the tool along with its version and any other useful identifying information, such as its locale.",
+        "startTime": {
+          "description": "The date and time at which the run started. See \"Date/time properties\" in the SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "endTime": {
+          "description": "The date and time at which the run ended. See \"Date/time properties\" in the  SARIF spec for the required format.",
+          "type": "string",
+          "format": "date-time"
+        },
+
+        "correlationId": {
+          "description": "An identifier that allows the run to be correlated with other artifacts produced by a larger automation process.",
           "type": "string"
         },
 
-        "version": {
-          "description": "The tool version.",
+        "architecture": {
+          "description": "The hardware architecture for which the run was targeted.",
           "type": "string"
         },
 
-        "semanticVersion": {
-          "description": "The tool version rendered as Semantic Versioning 2.0.",
+        "processId": {
+          "description": "The process identifier for the analysis tool run.",
           "type": "string"
         },
 
-        "fileVersion": {
-          "description": "The binary version of the tool's primary executable file (for operating systems such as Windows that provide that information).",
+        "workingDirectory": {
+          "description": "The working directory for the analysis rool run.",
           "type": "string",
-          "pattern": "[0-9]+(\\.[0-9]+){3}"
+          "default": { }
```

Comment:
No need for a default, but if there were one, it would have to be a `string` since that's the `type` of this property (so would be "`""`".